apps/docs/src/guide/errors/debugging-revert-errors.md

```diff
@@ -0,0 +1,27 @@
+# Debugging Revert Errors
+
+Unfortunately, the SDK does not support debugging revert errors with string messages - at least temporarily. This is because the SDK does not support decoding of Sway `str` slices in the `v0` Sway encoding scheme. This problem will be fixed soon once the `v1` Sway encoding scheme is adopted.
```

Comment:
@Dhaiwat10 If we tell users to use `__to_str_array()` it will be possible to decode the string error from the logs, like:

```rs
    fn test_function() -> bool {
        require(false, __to_str_array("This is a revert error"));
        true
    }
```

Reply:
this is also a "hack" because it won't work for slightly longer arbitrary-length strings. Per my observations, AbiEncoder/AbiDecoder within the Sway lib needs to implement the trait for str[32] onwards. 

may be slightly inaccurate with the numbers here...

Reply:
@theausicist In my tests, I was able to log a string with a length of 280:

<img width="271" alt="Screenshot 2024-03-13 at 16 05 12" src="https://github.com/FuelLabs/fuels-ts/assets/30977845/dd3d723b-e60e-4460-bd05-650486646ee2">

Everything works when decoding the log receipts:

<img width="809" alt="Screenshot 2024-03-13 at 16 11 20" src="https://github.com/FuelLabs/fuels-ts/assets/30977845/98aa9468-90af-41fb-9e02-160cd4b05019">

I don't see it as a hack since a `str[280]` is enough for an error message.

Reply:
Great suggestion @Torres-ssf

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1840/commits/2f5e137c64486b34a6f847b17fb0dd9afca5fdfa